string robot_name
---
string result